src/applications/vaos/containers/ExpressCareConfirmationPage.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect } from 'react';
+import { Link } from 'react-router';
+import { connect } from 'react-redux';
+import recordEvent from 'platform/monitoring/record-event';
+import { scrollAndFocus } from '../utils/scrollAndFocus';
+import { GA_PREFIX } from '../utils/constants';
+import {
+  startNewAppointmentFlow,
+  fetchFacilityDetails,
+} from '../actions/newAppointment';
+import { transformPendingAppointments } from '../services/appointment/transformers';
+import ExpressCareCard from '../components/ExpressCareCard';
+
+const pageTitle = 'You’ve successfully submitted your Express Care request';
+
+function ExpressCareConfirmationPage({ successfulRequest, router }) {
+  useEffect(
```

Comment:
This is essentially the same as `componentDidMount`, given that I'm including an array as the second arg and the params listed don't change.